graphql/src/store/resolver.rs

```diff
@@ -414,7 +413,7 @@ impl Resolver for StoreResolver {
             return self.lookup_meta(field).await;
         }
 
-        if object_type.is_sql() {
+        if ENV_VARS.graphql.enable_sql_service && object_type.is_sql() {
             return self.handle_sql(field);
         }
```

Comment:
the reason why we have this check here is in the remote case where the schema contains a type called `Sql` which a disabled SQL service graph-node would allow it.